### ISSUE TITLE
Remove JSON support

### ISSUE THREAD

**Issue by vswarte** [vswarte/gostatus#33]
Now YAML has been implemented for the configuration. I think it's safe to say we can get rid of JSON support.
YAML is more parseable for humans, it allows comments and it has anchors which make it less cumbersome to write config files.

**Comment by vswarte** [comment]
Example for all the YAML newcomers:
```
# Define a standard layout
layout: &BLOCK
    separator: true
    separatorBlockWidth: 15
    align: center

blocks:
    - <<: *BLOCK
      type: cpu_usage

    - <<: *BLOCK
      type: disk_usage

    - <<: *BLOCK
      type: time
      minWidth: 120
      format: Mon Jan 2 15:04:05
```

**Comment by vswarte** [comment]
Closed per #35
